viewing_party/party.py

```diff
@@ -1,23 +1,231 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+
+    #return None of title, genre, and rating are empty
+    if not (title and genre and rating):

```

Comment:
Nice way to check all three parameters without having to write `not` in front of each of them!